hs_tracking/middleware.py

```diff
@@ -0,0 +1,11 @@
+from .models import Session, Variable
```

Comment:
@ironfroggy Is the imported Variable module ever used? I don't see it being used currently  in this middleware module.